### PR TITLE
Bug 1411190, added information about init containers

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1411190

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape please peer review :eyes: 
@mfojtik PTAL

File: architecture/additional_concepts/admission_controllers.adoc [hunk 1]
```diff
@@ -121,3 +121,10 @@ ifdef::openshift-dedicated[]
 - xref:../../admin_guide/osd_build_defaults_overrides.adoc#admin-guide-osd-build-defaults-overrides[Configuring Global Build Defaults and Overrides]
 endif::[]
 - xref:../../admin_guide/scheduler.adoc#controlling-pod-placement[Controlling Pod Placement]
+
+[[admission-controllers-and-containers]]
+== Admission Controllers and Containers
```
Comment (bfallonf) [inline architecture/additional_concepts/admission_controllers.adoc]: @ahardin-rh I feel like this title is saying "Admission controllers and admission containers", but I don't think that's the meaning. Could it be something like "Using admission controllers in containers" or "Admission controllers that use containers"?

**Comment by ahardin-rh** [comment]
@bfallonf Thanks! Updated.

**Comment by mfojtik** [comment]
LGTM

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**